devices/Hts221/samples/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
guess what? better remove app.config... 😉

Reply:
good catch for all the PR. Removed as well the unnecessary version.json in samples and tests folders